<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFileWidget</class>
 <widget class="QWidget" name="NewFileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New File</string>
  </property>
  <property name="whatsThis">
   <string>Select the type of file you wish to create.&lt;br&gt;
&lt;p&gt;
&lt;b&gt;Pic Program&lt;/b&gt;&lt;br&gt;
Creates a new PIC program, with flow chart editor. Select the target device for your program below.
&lt;p&gt;
&lt;b&gt;Circuit&lt;/b&gt;&lt;br&gt;
Creates a new circuit, with drag and drop editor. Real time simulation of the circuit occurs automatically.</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="2" column="0" colspan="3">
    <widget class="MicroSelectWidget" name="m_pMicroSelect" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32767</width>
       <height>32767</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="textLabel1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>New File Details</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel1_2_2">
     <property name="text">
      <string>File Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="K3IconView" name="typeIconView"/>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="addToProjectCheck">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Add to project</string>
     </property>
     <property name="shortcut">
      <string>Alt+A</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3IconView</class>
   <extends>Q3Frame</extends>
   <header>q3iconview.h</header>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>K3IconView</class>
   <extends>Q3IconView</extends>
   <header>k3iconview.h</header>
  </customwidget>
  <customwidget>
   <class>MicroSelectWidget</class>
   <extends>QWidget</extends>
   <header>microselectwidget.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">microselectwidget.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
